src/views/shared/FooterHideableBackLink.js

```diff
@@ -0,0 +1,52 @@
+/*!
```

Comment:
Maybe we should rename this as `FooterForReset`, because this is for the password reset flow? 
I'm also not very sure we should even be creating a new footer. It is not extending an existing footer, so it seems like a lot of repeated code with just an extra if check. What do you think? @haishengwu-okta @nikhilvenkatraman-okta @mauriciocastillosilva-okta if you could weigh in as well

Reply:
Well, only one thing is really what we need to change from `FooterSignout` view is hiding the component if `hideBackToSignInForReset` is true. So if we can figure out how to make it easier than passing our FF to the template, then it'll be less repeatable. 

And also I believe we should move logic of click handling from the new component to `FooterSignout`. I mean this one:
![image](https://user-images.githubusercontent.com/41023010/54790102-a49d5080-4c3d-11e9-9a6b-3b7cede4249d.png)
Because, as I understand, we don't need to cancel a transaction if we don't have `settings.get('signOutLink')` property and just want to navigate back.

Reply:
I don't think we need a new footer at all, and also this is basically the same code as we have in `FooterSignout`.
The idea of `hideBackToSignInForReset` is to hide the sing out link, and we can do that by just **not adding** the footer in the controller(s). So the controller will have the logic to know if the footer is needed or not.